conf/charities.routes

```diff
@@ -3,4 +3,10 @@ GET        /                                                controllers.Charitie
 GET        /hello-world                                     controllers.HelloWorldController.helloWorld
```

Comment:
Standard is to have a prod.routes and an app.routes. I'm not too sure why we moved everything into a new routes file?